create_blog_post_using_azure_openai_and_publish_in_medium/main.bal

```diff
@@ -0,0 +1,45 @@
+import ballerina/io;
+import ballerinax/azure.openai.text;
+import ballerinax/medium;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string mediumToken = ?;
+
+const API_VERSION = "2023-03-15-preview";
+
+public function main(string title) returns error? {
+    final medium:Client medium = check new ({auth: {token: mediumToken}});
+    medium:UserResponse response = check medium->getUserDetail();
```

Comment:
```suggestion
    medium:UserResponse userResponse = check medium->getUserDetail();
```

Reply:
addressed in [d3600cd](https://github.com/ballerina-guides/ai-samples/pull/62/commits/d3600cd82ef0e11d2263647e00ce732672994597)